psa-ext/refval-meta.cddl

```diff
@@ -1,12 +1,17 @@
-$$reference-value-extension //= (
+$$reference-claim-map-extension //= (
   2 => psa-refval-meta
```

Comment:
On line 2 will it be cleaner to assign a code point rather than using value 2 like this?

Reply:
good catch!